core/trace/src/main/scala/org/typelevel/otel4s/trace/SpanOps.scala

```diff
@@ -56,7 +59,7 @@ trait SpanOps[F[_]] {
     * produce the effect, and ended when the effect completes.
     *
     * The finalization strategy is determined by [[SpanFinalizer.Strategy]]. By
-    * default, the abnormal termination (error, cancelation) is recorded.
+    * default, the abnormal termination (error, cancellation) is recorded.
```

Comment:
Cats-effect uses `cancelation` in the documentation, and we should too, I believe

Reply:
I don't have strong feelings about it (and it's trivial to revert the commit), but the spelling with a single 'l' is quite uncommon in American English, which tends to be what is used for spelling.

Reply:
Cats Effect used to use two l's.  I only know, because I corrected the inconsistencies back in CE2, when the spelling was more American than not.